stdlib/public/Cxx/std/String.swift

```diff
@@ -12,16 +12,19 @@
 
 extension std.string {
   public init(_ string: String) {
-    self.init()
-    for char in string.utf8 {
-      self.push_back(value_type(char))
+    let count = string.count
+    self.init(count, value_type(0))
+    var string = string
+    string.withUTF8 {
```

Comment:
Thanks for fixing this! 👍

Reply:
I thought this would be a small perf improvement but actually seems to not work. We can investigate in a separate follow up patch.